src/version.rs

```diff
@@ -0,0 +1,9 @@
+macro_rules! description {
```

Comment:
What's the benefit of using a macro here?

Reply:
Working around the fact that `concat` was brought to life without support for constants, ie. creating `pub const description: &'static str = "threescalers";` and doing `concat!(description, ...);` doesn't work as expected (ie. concat! uses literals only). There is demand to have it fixed or a new macro added, but that is still not done.

Reply:
OK :+1:  I didn't know that.